test/map_the_debris_test.js

```diff
@@ -0,0 +1,24 @@
+/*
+Tests for free code camp Map the Debris challenge
+https://www.freecodecamp.org/challenges/map-the-debris
+
+*/
+
+// testing framework, and assertion library
+var mocha = require('mocha');
+var expect = require('expect.js');
+
+// the thing we are testing
+var testFile = require('../challenges/start/map_the_debris__START.js');
+orbitalPeriod = testFile.orbitalPeriod;
+
+describe('Calculate the Orbital Period \n', function() {
+
+  it('"orbitalPeriod([{name : "sputnik", avgAlt : 35873.5553}])" \n \t should return \n \t \t [{name: "sputnik", orbitalPeriod: 86400}] \n \n', function() {
+    expect(orbitalPeriod([{name : "sputnik", avgAlt : 35873.5553}])).to.eql([{name: "sputnik", orbitalPeriod: 86400}]);
```

Comment:
looks like the same data is in the description and the test.  Is there a way to de-dupe this information?  Not a big deal...